src/python/pants/backend/native/config/environment.py

```diff
@@ -76,14 +79,21 @@ def as_invocation_environment_dict(self):
     })
     return {
       'PATH': create_path_env_var(self.path_entries),
-      lib_env_var: create_path_env_var(self.library_dirs),
+      lib_env_var: create_path_env_var(self.runtime_library_dirs),
     }
 
+  def as_execute_process_request(self, more_args=None):
+    argv = [self.exe_filename] + self.extra_args + (more_args or [])
```

Comment:
`more_args` can be a tuple here to eliminate the `or`

Reply:
Only sort-of. If you want to be defensive, this is appropriate. The caller can explicitly pass `None`.